[WIP] Add initial decision making process to community handbook (#3940)
the-turing-way/the-turing-way: book/website/community-handbook/working-groups/working-groups-formalisation.md

Please reach out to them if you have any questions about the process or would like support in getting started.

A WG may focus on:
1. A specific topic related to or of interest to *The Turing Way* community.
   * For example: localisation, professionalising research infrastructure roles, accessibility in open source communities and outputs.
2. A type of ongoing task or work currently being done within the project.
   * For example: Book Dash event strategy and delivery, open-source infrastructure maintenance, providing training and mentorship.
3. Any other aspect you would like to introduce to the community! `for discussion - is this true?`

### Proposing a Working Group

A short proposal should be created and shared as an issue on *The Turing Way*'s main GitHub repository. 
This proposal doesn't have to be exhaustive but must outline the area of work being proposed and where needs or opportunities have been identified in the project, community, or within the broader open science or research ecosystem of *The Turing Way*. 
Please identify and include at least 3 to 4 members who are interested in participating and contributing to the proposed area of work.

> Ideally, the proposing party should discuss their plan with *The Turing Way* Research Community Manager or invite support from other members of the project delivery team.

Reviewer: Think we need to re-think this - honestly I don't see the need for an intermediary here and we have multiple project delivery teams now existing within the Turing Way, so I think we need to update this. 

My suggestions is: We should invite proposals to be posted on GitHub, invite community comments in Slack and via the newsletter, and community discussions at Collaboration Cafes and the Community Forums, with the Steering Committee ultimately giving the thumbs up. 

If the timeline is too short for set up (say someone wants to apply to an event as part of the Working Group with a deadline in the next month) then we can think about how to streamline approval or maybe give retroactive approval from the Steering Committee.

    * Note: In "WG Resources," other infrastructure recommendations and needs (permissions, templates, and checklists) are available to help with this process.
* **Establish a Reporting Mechanism and/or Public Notes Archive**: How will you share meeting notes and group progress with the wider community?
    * Note: All WGs should upload notes to their respective GitHub repository [in *The Turing Way* organisation](https://github.com/the-turing-way/the-turing-way/). For a template, please see this template repository.
* **Establish Appropriate Permissions for Accounts**: What accounts does this group need access to? Does this group have the proper account access and permissions to carry out its work?
    * Note: The Community Manager and/or Project Manager can assist with this!
* **Create a WG Charter**: Please follow the {ref}`Working Groups Charter template<ch-gov-wg-charter-template>`.
    * Note: All WGs should openly share an updated charter to their respective GitHub repository [in *The Turing Way* organisation](https://github.com/the-turing-way/the-turing-way/).

> * **Create an Open Calendar Invite**: This may require support from a core delivery staff member to add the information to the public Google Calendar.

Reviewer: ```suggestion
* **Create an Open Calendar Invite**: This may require support from a member of the Community Management Working Group to add the information to the public Google Calendar.
```

* **Advertise on Community Channels**: Use Slack, GitHub, social media, and/or the *The Turing Way* newsletter to share information about your WG.

>     * Note: The Community Manager and/or Project Manager can assist with this! Ask them on Slack or tag them on GitHub to share out this information, or co-create a communication plan.

Reviewer: ```suggestion
    * Note: The Community Management Working Group can assist with this! Ask them on Slack or tag them on GitHub to share out this information, or co-create a communication plan.
```

* **Create a Slack Channel for Collaboration** (not required but encouraged): Having an ongoing channel makes communication easier. What other communication channels might this group need?

> * **Add Information about WG in Ways of Working and/or Governance Documents**: Is there a public and up-to-date record of this WG?

Reviewer: ```suggestion
* **Add Information about Working Group in Ways of Working and/or Governance Documents**: Is there a public and up-to-date record of this Working Group in the project's central documents?
```

### Maintaining a Working Group

> While initiating a WG, maintaining its activities is integral for the group's success. 

Reviewer: ```suggestion
While initiating a Working Group, maintaining its activities is integral for the group's success. 
```

On a quarterly basis, a WG should make time to revisit its structures and ways of working to ensure it is adhering to its established purpose. 

The following questions can guide this process:

* **Review Purpose**: Is this WG still fulfilling its established purpose?

> * **Review Progress**: Has this WG been able to make progress towards its stated goals? What adjustments may need to be made to support this work?

Reviewer: ```suggestion
* **Review Progress**: Has this Working Group been able to make progress towards its stated goals? What adjustments, if any, should be made to support this work?
```

* **Review Process**: Is the existing way of working valuable for this team currently? Are there other processes (decision-making, conflict management) that may need adjustment?

>     * Note: The Community Manager and/or Project Manager can assist with facilitating this!

Reviewer: ```suggestion
    * Note: The Community Management Working Group can assist with facilitating this!
```
